tests/dir.props

```diff
@@ -29,14 +32,23 @@
     <PackagesDir>$(ProjectDir)..\packages\</PackagesDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)..\Tools\</ToolsDir>
     <DotnetCliPath Condition="'$(DotnetCliPath)'==''">$(ToolsDir)dotnetcli/</DotnetCliPath>
+    <OverrideToolHost>$(DotnetCliPath)dotnet</OverrideToolHost>
     <BuildToolsTaskDir Condition="'$(BuildToolsTargets45)' == 'true'">$(ToolsDir)net46/</BuildToolsTaskDir>
+    <CSharpCoreTargetsPath Condition="'$(OS)' != 'Windows_NT'">$(ToolsDir)\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
+    <_TargetFrameworkDirectories Condition="'$(OS)' != 'Windows_NT'">$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
```

Comment:
Please also copy the [comment](https://github.com/dotnet/corefx/blob/16625287b450bc623fe46683f0ff27da1ab5ad47/dir.targets#L115-L116) from the corefx equivalent to make this more discoverable.